script/circleci

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+case $CIRCLE_NODE_INDEX in
+  0 )
+    bundle exec rake test
```

Comment:
Also note to self:  the `bundle exec rake test` duplication can easily be solved with `*)`